ultraThreshold.py

```diff
@@ -34,7 +37,7 @@
     uhal.setLogLevelTo( uhal.LogLevel.ERROR )
 
 import ROOT as r
-filename = options.filename
+filename = options.filename + "_shelf%i_s%i_g%i.root"%(options.shelf,options.slot,options.gtx)
```

Comment:
This will not play nice with current infrastructure, where the structure is organized from top level as a given chamber and down.
Please revert for now, discussion will continue in #90